src/libs/ajax/data-table-providers/DataTableProvider.ts

```diff
@@ -50,6 +50,36 @@ export interface EntityQueryOptions {
   filterOperator: string
 }
 
+export type UploadParameters = {
+  file: File
+  useFireCloudDataModel: boolean
+  deleteEmptyValues: boolean
+  namespace: string
+  name: string
+  workspaceId: string
+  recordType: string
+}
+
+export type isInvalidTsvOptions = {
```

Comment:
Conventionally, type names are capitalized.

Reply:
I'm used to seeing names that start with `is` be used for variables/functions that hold/return a boolean value. I wonder if it could be `InvalidTsvOptions` or such.